tests/data/product_stability/ubuntu2204.yml

```diff
@@ -1,4 +1,4 @@
-aide_also_checks_audispd: 'yes'
+aide_also_checks_audispd: 'no'
```

Comment:
Looks like there is a difference between CIS and STIG with respect to this variable.
According to Ubuntu 22.04 STIG, rule UBTU-22-651030, aide should also check the integrity of `audispd`.

I suggest we re-set the var to `yes`, enabling the check for all benchmarks, and thus introducing an (negligible) extra check in CIS, instead of having a missing check in STIG.

Reply:
@mpurg audispd doesn't exist on 22.04. DISA need to update their benchmark.

Reply:
Ooops, mybad, should've read the original issue 🤦 
All good then, I'll inform DISA.